runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -438,11 +602,37 @@ else if (!isSkill || exp != -1L)
 
 				if (level != -1)
 				{
-					label.setText(Integer.toString(level));
+					// Format numbers to prevent overflowing the labels
```

Comment:
Use one of the standard text formatters for this

Reply:
The available formatters dont allow the type of formatting required, in some situations the numbers would be displayed partially